specification/dataprotection/resource-manager/Microsoft.DataProtection/stable/2023-01-01/examples/ResourceGuardProxyCRUD/PutResourceGuardProxy.json

```diff
@@ -0,0 +1,38 @@
+{
+  "parameters": {
+    "subscriptionId": "5e13b949-1218-4d18-8b99-7e12155ec4f7",
+    "vaultName": "sampleVault",
+    "resourceGroupName": "SampleResourceGroup",
+    "resourceGuardProxyName": "swaggerExample",
+    "api-version": "2023-01-01",
+    "parameters": {
+      "properties": {
+        "resourceGuardResourceId": "/subscriptions/f9e67185-f313-4e79-aa71-6458d429369d/resourceGroups/ResourceGuardSecurityAdminRG/providers/Microsoft.DataProtection/resourceGuards/ResourceGuardTestResource"
+      }
+    }
+  },
+  "responses": {
```

Comment:
>responses [](http://example.com/codeflow?start=3&length=9)

nit, I'd suggest you should rename this example either 'create' or 'update', or createorupdate, so its obvious which scenario it is. And add the other scenario also.